engine/src/eps.cpp

```diff
@@ -64,6 +64,7 @@ MCEPS::MCEPS()
 	pagecount = 0;
 	curpage = 1;
 	pageIndex = NULL;
+    setparent(MCdefaultstackptr);
```

Comment:
I'm pretty sure that no other constructors set their parent in the default constructor - setparent should be called before import() is in the exec handler I think...